db/migrate/20231027162850_create_orders.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
t.references :user, null: false, foreign_key: { on_delete: :cascade }
 
 Sugestão:
 A adição do foreign_key: { on_delete: :cascade } dentro da migração na definição da chave estrangeira t.references :user garante que, se o usuário associado a um pedido for excluído, os pedidos relacionados também serão automaticamente excluídos, mantendo a integridade referencial.